packages/buffetjs-styles/src/components/TimeList/index.js

```diff
@@ -60,10 +60,10 @@ const TimeList = styled.ul`
       z-index: 0;
       width: 100%;
       height: 100%;
-      padding: ${sizes.margin * 0.3}px 0 ${sizes.margin * 0.3}px
+      padding: ${sizes.margin * 0.4}px 0 ${sizes.margin * 0.2}px
```

Comment:
Why do we have a mix between `rem` and `px` here? Also it might be great if the `sizes` file includes the units completely or does not at all. Maybe we should make it uniform in another PR.

Reply:
For now, the sizes values are made for px. But I agree, we should uniform all the units and sizes soon in another PR!